reference/api/tasks.md

```diff
@@ -0,0 +1,192 @@
+# Tasks
+
+The `tasks` route gives information about the progress of [asynchronous operations](/learn/advanced/asynchronous_operations.md).
+
+:::note
+The task `uid` is incremented **globally.**
+:::
+
+## Get all tasks
+
+<RouteHighlighter method="GET" route="/tasks"/>
+
+List all tasks globally regardless of the index involved. The `task` objects are contained in the `results` array.
+
+### Example
+
+<CodeSamples id="get_all_tasks_1" />
+
+#### Response: `200 Ok`
+
+```json
+{
+    "results": [
+        {
+            "uid": 1,
+            "indexUid": "movies_reviews",
+            "status": "enqueued",
+            "type": "documentsAddition",
+            "duration": null,
+            "enqueuedAt": "2021-08-12T10:00:00.000000Z",
+            "startedAt": null,
+            "finishedAt": null
+        },
+        {
+            "uid": 0,
+            "indexUid": "movies",
+            "status": "succeeded",
+            "type": "documentsAddition",
+            "details": { 
+                    "receivedDocuments": 100,
+                    "indexedDocuments": 100
+            },
+            "duration": "PT16S",
+            "enqueuedAt": "2021-08-11T09:25:53.000000Z",
+            "startedAt": "2021-08-11T10:03:00.000000Z",
+            "finishedAt": "2021-08-11T10:03:16.000000Z"
+        }
+    ]
+}
+```
+
+:::note
+By default, **task unique identifiers (`uid`) are sorted in `desc` order.** This means that the most recent `task` objects appear first.
```

Comment:
Removed "by default" because I don't believe there's any way to alter that behavior.

```suggestion
Tasks are displayed in descending order by `uid`. This means that **the most recently created `task` objects appear first**.
```

Reply:
I think in most cases, sorting by `uid` should be the same as sorting by `enqueuedAt`—it's the order in which requests were received. I wonder if there's a case where this is not true? 🤔

@gmourier tagging you because I'm curious; it's not important so please don't feel any pressure to respond 😇